.github/workflows/fuzzing-ci.yaml

```diff
@@ -56,15 +56,6 @@ jobs:
         source ./ci/common.sh
         export_version_info
         export_or_prefix
-        wget -qO - https://openresty.org/package/pubkey.gpg | sudo apt-key add -
-        sudo apt-get update
-        sudo apt-get -y install software-properties-common
-        sudo add-apt-repository -y "deb http://openresty.org/package/ubuntu $(lsb_release -sc) main"
-        sudo apt-get update
-        sudo apt-get install -y git curl openresty-openssl111-dev unzip make gcc libldap2-dev
-        ./utils/linux-install-luarocks.sh
-
-        make deps
```

Comment:
Why do you remove this?

Reply:
They are finished in `do_install` and `before_install`.